### PR TITLE
Create glossary for node

### PR DESCRIPTION
**Pull request by spzala**
Add node into the glossary.

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.9 Features: set Milestone to `1.9` and Base Branch to `release-1.9`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.
>
> NOTE: After opening the PR, please *un-check and re-check* the "Allow edits from maintainers" box. This is a temporary workaround to address a known issue with GitHub.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6573)

### CODE REVIEW

File: _data/glossary/node.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+full-link: /docs/concepts/architecture/node
+tags:
+- cluster
+- deployment
+- fundamental
+- pod
+short-description: >
+  A node is a worker machine in Kubernetes.
+long-description: >
+  A node may be a VM or physical machine, depending on the cluster.
```
Comment (zhangxiaoyu-zidif) [inline _data/glossary/node.yaml]: I remember that even a kubernetes pod can act as a node, such as in kubernetes in kubernetes architecture. so is that necessary to add pod as a node object? :)

**Comment by Bradamant3** [comment]
Hi @spzala -- this looks good. While we're waiting on the Travis build to pass, though -- you could do a bit more with the long description -- maybe list the processes that run on a node, explain that nodes are managed by the master.

**Comment by chenopis** [comment]
Here's the Travis error:
> --- FAIL: TestCanonicalTags (0.00s)
	glossary_test.go:101: Glossary term "Node" has invalid tag "cluster". See ../_data/canonical-tags for the list of valid tags.
	glossary_test.go:101: Glossary term "Node" has invalid tag "deployment". See ../_data/canonical-tags for the list of valid tags.
	glossary_test.go:101: Glossary term "Node" has invalid tag "pod". See ../_data/canonical-tags for the list of valid tags.

**Comment by spzala** [comment]
@chenopis  thanks for explaining tags and differences between tags and related!!

**Comment by spzala** [comment]
@Bradamant3 thanks for the comment, so we have a good long page description for node https://kubernetes.io/docs/concepts/architecture/nodes/ and I have added it as a full link. So thought don't need more in the long description. But let me add some more description. Thanks!

**Comment by spzala** [comment]
@Bradamant3 Thanks!! :-)

**Pull request merged**